testsuite/test-depmod.c

```diff
@@ -89,14 +89,18 @@ DEFINE_TEST(depmod_search_order_simple,
 	});
 
 #define ANOTHER_MODDIR "/foobar"
+#define RELATIVE_MODDIR "foobar2"
 #define MODULES_ANOTHER_MODDIR_ROOTFS TESTSUITE_ROOTFS "test-depmod/another-moddir"
-#define MODULES_ANOTHER_MODDIR_LIB_MODULES \
-	MODULES_ANOTHER_MODDIR_ROOTFS ANOTHER_MODDIR "/" MODULES_UNAME
```

Comment:
is  this comment in the right line? MODULES_NAME is not being removed here.

Reply:
I don't know what github did... but it showed this comment in the previous commit where MODULES_UNAME was not being revoved. Yes, it was intentional because I added it in the `.files = ...`